### PR TITLE
Fixed trigger attribution URL parameter

### LINKED ISSUE #168: Fix the trigger attribution .well-known URL
In the [event-level click proposal](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#triggering-attribution), the .well-known URL is `trigger-attribution?data=....`. But the correct URL seems to be `trigger-attribution?trigger-data=...`

* What's been implemented in Chrome 92 is `trigger-data` as can be sen in [this test](content/browser/conversions/conversion_registration_browsertest.cc)
* [PCM](https://privacycg.github.io/private-click-measurement/#scenario) seems to refer to `trigger-data`, too

### PR DESCRIPTION
**Pull request by maudnals**
Fixes #168

### CODE REVIEW

**Comment by maudnals** [comment]
@johnivdel Can you PTAL? Thanks!

**Review by apasel422 — APPROVED** [review]

File: event_attribution_reporting_clicks.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ this API:
 to trigger attribution for all matching sources.
 
 The browser will treat redirects to a URL of the form:
-`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?data=<data>&priority=<priority>]`
+`https://<attributionreportto>/.well-known/attribution-reporting/trigger-attribution[?trigger-data=<trigger-data>&priority=<priority>]`
```
Comment (johnivdel) [inline event_attribution_reporting_clicks.md:194]: Can we update the name on row 198?

**Review by johnivdel — COMMENTED** [review]
lgtm % comment

File: event_attribution_reporting_clicks.md (hunk 1 above)
Comment (maudnals) [reply event_attribution_reporting_clicks.md:194]: Thanks for catching this! Fixed row 198.

**Review by johnivdel — APPROVED** [review]

**Comment by apasel422** [comment]
@maudnals There are some conflicts in this PR that need to be resolved before we can submit this.

**Pull request closed without merging**